src/components/card/Card.jsx

```diff
@@ -1,82 +1,79 @@
 
 import React, { Component } from 'react'
 import PropTypes from 'prop-types'
-import { Layout, Button, Select } from 'antd'
+import { Layout } from 'antd'
 import './style.css'
 import EditArea from '../../commons/editArea/EditArea'
+import Loader from '../../commons/loader/Loader'
+import LabelsContainer from '../../containers/labels/LabelsContainer'
+import AssigneesContainer from '../../containers/assignees/AssigneesContainer'
 
-const { Sider, Content } = Layout
-
-const Option = Select.Option
+const { Content } = Layout
 
 
 class Card extends Component {
   constructor(props) {
     super(props)
-    this.handleChangeAssignee = this.handleChangeAssignee.bind(this)
-    this.handleChangeLabels = this.handleChangeLabels.bind(this)
+    this.getLabels = this.getLabels.bind(this)
+    this.getAssignees = this.getAssignees.bind(this)
   }
 
-  handleChangeAssignee(value) {
-    const memberId = value[value.length - 1].slice(this.props.id.length, value.toString().length)
-    if (this.props.card.assignees.map(element => element.id).includes(memberId) === true) {
-      this.props.removeAssigneeInCard(this.props.card.id, memberId)
-    } else {
-      this.props.addAssignee(this.props.card.id, memberId)
-    }
+  getAssignees() {
+    return (
+      <div>
+        { this.props.card.isFetchingAssignees ? (
+          <Loader message="Loading labels..." />
+        ) : (
+          <AssigneesContainer cardId={this.props.id} displayAssignees displaySelect />
+        )}
+      </div>
+    )
   }
 
-  handleChangeLabels(value) {
-    const labelId = value[value.length - 1].slice(this.props.id.length, value.toString().length)
-    if (this.props.card.labels.map(element => element.id).includes(labelId) === true) {
-      this.props.removeLabelInCard(this.props.card.id, labelId)
-    } else {
-      this.props.addLabel(this.props.card.id, labelId)
-    }
+  getLabels() {
+    return (
+      <div>
+        { this.props.card.isFetchingLabels ? (
+          <Loader message="Loading labels..." />
```

Comment:
Why do you fetch labels here ? I think you just need to use the container and let it to handle such things (including the loading). So you don't need a `getLabels` function, just call `<LabelsContainer ... />` in the render method.